include/RAJA/util/Operators.hpp

```diff
@@ -386,7 +386,7 @@ struct logical_and : public detail::comparison_function<Arg1, Arg2>,
   {
     return lhs && rhs;
   }
-  static constexpr const bool identity = true;
+  static constexpr bool identity() {return true;}
```

Comment:
These were set up by @willkill07 with the operators in the first place.  They provide a default identity value for a given operator, which can be very useful.  I made this change because for some reason the constexpr values weren't being inlined in some cases, and weren't being built into the final binary either.  The function versions get generated correctly however.